ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -2329,18 +2318,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-x86-perfscale-rosa-hcp-3zones-cluster-density-v2-24nodes-periodic
-  reporter_config:
```

Comment:
Let's verify that each of the tests has this "reporter_config" under the name in the periodic job file

Reply:
Ack, added!